tests/integration/routes/account/patch-account-test.js

```diff
@@ -45,10 +45,14 @@ var routeOptions = {
   }
 }
 function responseMock () {
-  // user document gets fetched twice, once for session.find(),
-  // then for account.update(). We might improve that by implementing
+  // user document gets fetched 3 times:
+  // 1) session.find()
+  // 2) account.update()
+  // 3) session.add()
```

Comment:
I'm a little confused because I had to add the third call to `mockUserFound`. see 3 lines below, we call it 3 times (the last time in the return statement). Maybe I'm missing a detail???